spec/inputs/gelf_spec.rb

```diff
@@ -1,8 +1,22 @@
 require "logstash/devutils/rspec/spec_helper"
 require "logstash/inputs/gelf"
+require_relative "../support/helpers"
 require "gelf"
 
-describe "inputs/gelf" do
+Thread.abort_on_exception = true
+describe LogStash::Inputs::Gelf do
+  context "when interrupting the plugin" do
+    let(:port) { rand(1024..65535) }
```

Comment:
Some plugins might not want this for some reason, so the right way is to make this configurable somehow. The problem is that Rspec.configuration block is global :(